### DISCUSSION TITLE
Using Windows 10 for the script

### DISCUSSION THREAD

**Discussion by sarmisthan** [biostars/biostar-handbook-issues#203] [Q&A]
Hello,

Can I use all the scripts in Windows 10? If so how?

Thanks Sramistha

**Comment by ialbert** [comment]
Yes, the book has been designed with Windows in mind, about half the students every year use Windows.

You will need to use the Linux Subsystem for Windows. Thankfully installing that is a fairly straightforward process.

The installation chapter describes the Windows installation process in great detail.
